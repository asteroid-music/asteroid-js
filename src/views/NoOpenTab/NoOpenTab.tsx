//Import from external 'react' module
import React from 'react';

//Import from external '@material-ui/core' module
import Typography from '@material-ui/core/Typography';

/**
 * React component for the main body of the page with no open tab
 */
function NoOpenTab(props: {}) {
    return <Typography>No tab selected!</Typography>
}

export default NoOpenTab;
