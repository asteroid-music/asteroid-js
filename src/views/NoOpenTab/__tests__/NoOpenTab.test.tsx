import React from 'react';
import renderer from 'react-test-renderer';
import NoOpenTab from 'views/NoOpenTab';

describe('<NoOpenTab />',()=>{
    //Basic rendering with minimal props
    describe('basic rendering',()=>{
        it("renders correctly",()=>{
            expect(renderer
                .create(<NoOpenTab/>)
                .toJSON()
            ).toMatchSnapshot();
        });
    });
});