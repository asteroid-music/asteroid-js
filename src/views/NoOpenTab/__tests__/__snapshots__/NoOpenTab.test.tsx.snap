// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NoOpenTab /> basic rendering renders correctly 1`] = `
<Material UI Container>
  <Material UI Typography>
    No tab selected!
  </Material UI Typography>
</Material UI Container>
`;
